.github/workflows/test.yml

```diff
@@ -0,0 +1,198 @@
+name: test
```

Comment:
typically this should be more descriptive than "test" this is I believe the friendly name that will show up in the gui, something like "omnitest-workflow" that is more descriptive would be great.

Reply:
There are various views so yeah, there's probably somewhere this would be annoying.  On the flip side, I was getting tired of the job lists looking like `MacOS blockchain Tests / MacOS blockchain Tests (3.8, macOS-latest) (pull_request)` vs. `test / 🍎 clvm - 3.8 (pull_request) `.

What particular place do you think `test` is insufficient so I can see what we need to clarify it against?

Reply:
Since the final job name ends up being something like `test / 🍎 clvm - 3.8 (pull_request)` - I think this looks ok to me. Easier to read at a glance and doesn't get cut off with `...` most places